sql-plugin/src/main/scala/com/nvidia/spark/rapids/basicPhysicalOperators.scala

```diff
@@ -381,6 +381,7 @@ case class GpuSampleExec(lowerBound: Double, upperBound: Double, withReplacement
 
   override lazy val additionalMetrics: Map[String, GpuMetric] = Map(
     OP_TIME -> createNanoTimingMetric(MODERATE_LEVEL, DESCRIPTION_OP_TIME))
+  // TODO CPU vs GPU OP TIME in Debug mode???
```

Comment:
Something missing here or is this a followup issue?

Reply:
Sorry no the follow on was already done. I just ran into some things with sample when I tried to use it for testing and ended up debugging too much on this branch.